AveryLabels.py

```diff
@@ -127,4 +129,4 @@ def render_iterator( self, func, iterator ):
                 canv.rect( 0, 0, self.size[0], self.size[1] )
             func( canv, self.size[0], self.size[1], chunk )
             canv.restoreState()
-            self.advance()
\ No newline at end of file
+            self.advance()
```

Comment:
Please restore the blank line at the end. As more people are contributing, I will add an `.editorconfig` soon.
```suggestion
            self.advance()

```